easybuild/easyblocks/n/neuron.py

```diff
@@ -60,85 +62,138 @@ def extra_options():
         extra_vars = {
             'paranrn': [True, "Enable support for distributed simulations.", CUSTOM],
         }
-        return ConfigureMake.extra_options(extra_vars)
+        return CMakeMake.extra_options(extra_vars)
 
     def configure_step(self):
         """Custom configuration procedure for NEURON."""
+        if LooseVersion(self.version) < LooseVersion('7.8.1'):
```

Comment:
@mboisson There's a small correction needed here to avoid that `cmake` is used as configure command for older NEURON versions, fixed in https://github.com/ComputeCanada/easybuild-easyblocks/pull/10

With this fix included, installing the existing NEURON easyconfigs (7.4 with `intel/2016b` and 7.6.5 with `intel/2018b`) works fine, as does 7.8.2 with `foss/2021b` (so using CMake)